cmd/entrypoint/main.go

```diff
@@ -33,9 +33,9 @@ import (
 	featureFlags "github.com/tektoncd/pipeline/pkg/apis/config"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
 	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
-	"github.com/tektoncd/pipeline/pkg/credentials"
 	"github.com/tektoncd/pipeline/pkg/credentials/dockercreds"
 	"github.com/tektoncd/pipeline/pkg/credentials/gitcreds"
```

Comment:
We will need to do the same for `credentials/{dockercreds,gitcreds…` 👼🏼

Reply:
not sure what you mean 🤔 😅

Reply:
Not necessarily not use them, but finding a way so they do not bring `corev1` with it.